livedata/src/main/java/io/getstream/chat/android/livedata/entity/ChannelEntity.kt

```diff
@@ -105,9 +118,17 @@ data class ChannelEntity(var type: String, var channelId: String) {
         return c
     }
 
-    /** updates last message and lastmessagedate on this channel entity */
-    fun updateLastMessageDate(messageEntity: MessageEntity) {
-        lastMessageAt = max(lastMessageAt, messageEntity.createdAt ?: messageEntity.createdLocallyAt)
+    /** updates last message and lastMessageAt on this channel entity */
+    fun addMessage(messageEntity: MessageEntity) {
```

Comment:
This method name is a bit confusing. It is not adding any message to any part

Reply:
renamed it to updateLastMessage